graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -477,6 +481,13 @@ class RenameSchemaTypesVisitor(Visitor):
             "UnionTypeDefinitionNode",
         }
     )
+    name_conflicts: Dict[str, Set[str]]  # Collects naming conflict errors. If renaming
+    # would result in multiple types being renamed to "Foo", name_conflicts will map "Foo" to an
+    # set containing the name of each such type.
+    renamed_to_scalar_conflicts: Dict[str, str]  # Collects attempts to rename a type with a name
+    # already used by a scalar. If renamings tries to rename a type named "Foo" to "Bar" and there
+    # already exists a scalar named "Bar" in the schema, then
+    # renamed_to_scalar_conflicts["Foo"] == "Bar"
```

Comment:
any reason to not include `reverse_name_map` up here rather than in the `__init__`? might be good to add typing/descriptions for the other class properties as well.

Reply:
Done for `reverse_name_map`! My rule for these is if an attribute gets used outside of the class, I make it a class attribute, so we can leave the others as-is.